core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -190,7 +190,7 @@ class KafkaApis(val requestChannel: RequestChannel,
         case ApiKeys.HEARTBEAT => handleHeartbeatRequest(request)
         case ApiKeys.LEAVE_GROUP => handleLeaveGroupRequest(request)
         case ApiKeys.SYNC_GROUP => handleSyncGroupRequest(request, requestLocal)
-        case ApiKeys.DESCRIBE_GROUPS => handleDescribeGroupRequest(request)
+        case ApiKeys.DESCRIBE_GROUPS => handleDescribeGroupsRequest(request)
```

Comment:
I see we changed this to be plural -- was the reasoning because of the API?

Reply:
That's right.